socrates/test/activities/socratesActivitiesService_test.js

```diff
@@ -95,19 +95,18 @@ describe('SoCraTes Activities Service', function () {
         expect(newParticipantNotification.called).to.be.true();
 
         const readModel = cache.get(socratesConstants.currentUrl + '_registrationReadModel');
-        expect(readModel.reservationsAndParticipantsFor('single')).to.have.length(1);
 
         done(err);
       });
     });
 
-    it('registers the user even when he is not on the waitinglist', function (done) {
+    it('does not register the user when he is not on the waitinglist', function (done) {
 
       socratesActivitiesService.fromWaitinglistToParticipant({nickname: 'nickname', roomType: 'single', duration: 2}, now, function (err) {
         expect(stripTimestamps(saveEventStore.firstCall.args[0].state.events)).to.eql([
-          {event: e.PARTICIPANT_WAS_REGISTERED, sessionId: undefined, roomType: 'single', memberId: 'memberId', duration: 2, joinedSoCraTes: now.valueOf()}]);
+          {event: e.DID_NOT_REGISTER_PARTICIPANT_FROM_WAITINGLIST_BECAUSE_THEY_WERE_NOT_ON_WAITINGLIST, roomType: 'single', memberId: 'memberId', duration: 2}]);
```

Comment:
Das ist pro übermitteltem ausgewähltem Zimmer. Wenn das Frontend eine Einbuchung von der Warteliste in ein Zimmer übermittelt, aber das Backend den Wartelisten-Eintrag nicht findet, dann wurde der Benutzer bisher trotzdem eingebucht. Das ist aber Quatsch, weil das nicht vorkommen kann in der normalen Nutzung (woher soll das Frontend die Infos haben, wenn nicht aus einem Wartelisten-Event?), also deutet das entweder auf einen Bug oder einen Missbrauch der API hin. Da ich das PARTICPANT_WAS_REGISTERED Event sowieso loswerden wollte, hab ich das so umgebaut. - Hat keine inhaltliche Bedeutung, dient nur zum Festhalten der übermittelten Argumente.